pages/osx/timedatectl.md

```diff
@@ -0,0 +1,12 @@
+# timedatectl
+
+> Show or control the system date and time.
+> More information: <https://www.freedesktop.org/software/systemd/man/timedatectl.html>.
+
+- Show current time/date, and options:
```

Comment:
```suggestion
- Print the current data, time and more information:
```

Reply:
probably we meant `date` and not `data` :)